WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -454,6 +452,7 @@ - (void)viewDidAppear:(BOOL)animated
 - (void)viewWillDisappear:(BOOL)animated
 {
     [super viewWillDisappear:animated];
+    [self stopObservingWillEnterForegroundNotification];
```

Comment:
We don't want to observe this notification if this screen is not visible. If the user navigates back to this screen, the `configureTableViewData` is called on `viewWillAppear`.